features/being_notified_about_changes_in_order.feature

```diff
@@ -0,0 +1,67 @@
+@reordering
+Feature: Being notified about changes in order
+    In order to be aware of disabled promotions or changed order total
+    As a Customer
+    I want to be notified about such changes when I reorder previously placed order
+
+    Background:
+        Given the store operates on a single channel in the "United States" named "Web"
+        And the store has a product "Angel T-Shirt" priced at "$39.00"
+        And there are 25 units of product "Angel T-Shirt" available in the inventory
+        And this product is tracked by the inventory
```

Comment:
I guess that there is no need for this step, because the product is set as tracked in the previous step